webrender/src/render_backend.rs

```diff
@@ -214,6 +214,10 @@ impl RenderBackend {
                             }
                         }
                         ApiMsg::SetRootPipeline(pipeline_id) => {
+                            if scene.pipeline_map.get(&pipeline_id).is_none() {
```

Comment:
Edit: I'm wrong, the pipeline is also set up in `set_root_display_list`. Perhaps, it would be cleaner (and less confusing) to check for the display list here?